image-filters/image-filters.markdown

```diff
@@ -1064,7 +1063,7 @@ public class ColorHelper {
 ```
 
 Clearly we can’t test this with whole images. Instead we can mock the images
```

Comment:
Actually when I ported this to Obj-C, I did make something that would make actual images with certain properties and allow me to compare them, so they didn't have to be mocked. https://github.com/catehstn/image-creator